libraries/c_sdk/standard/ble/src/services/mqtt_ble/iot_ble_mqtt_serialize.c

```diff
@@ -38,17 +38,34 @@
 /* FreeRTOS includes. */
 #include "FreeRTOS.h"
 #include "iot_ble_config.h"
-#include "core_mqtt_serializer.h"
 
 /* MQTT internal includes. */
-#include "platform/iot_threads.h"
 #include "iot_serializer.h"
 #include "iot_ble_data_transfer.h"
 #include "iot_ble_mqtt_serialize.h"
-#include "logging_stack.h"
 
-#define _INVALID_MQTT_PACKET_TYPE    ( 0xF0 )
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Include logging header files and define logging macros in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define the LIBRARY_LOG_NAME and LIBRARY_LOG_LEVEL macros depending on
+ * the logging configuration for DEMO.
+ * 3. Include the header file "logging_stack.h", if logging is enabled for DEMO.
+ */
+
+#include "logging_levels.h"
```

Comment:
The logging configuration should _ideally_ be exposed in a config file for the library so that the user can update the configuration from the project files under `aws_demos/config_files` directory of the vendor. 

Is there a ble_config.h file that can be utilized for storing the logging configuration?
This change can be made in a separate PR though to get these changes in for the FreeRTOS console testing.

Reply:
Yes, I can refactor the config into a different file in a separate PR, Probably not `iot_ble_config.h` as it contains the config for bluetooth module.